Sources/_StringProcessing/Regex/Match.swift

```diff
@@ -17,17 +17,13 @@ extension Regex {
   /// providing direct access to captures.
   @dynamicMemberLookup
   public struct Match {
-    let input: String
+    let anyRegexOutput: AnyRegexOutput
 
     /// The range of the overall match.
     public let range: Range<String.Index>
```

Comment:
Doesn't ARO store this?

Reply:
The ARO inside of Match does not store it, but match.output where Output == ARO does store it.

Reply:
Wait, so sometimes ARO stores it and sometimes it doesn't? That seems like a footgun.